var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -358,6 +358,9 @@ class Llvm(CMakePackage, CudaPackage):
     # avoid build failed with Fujitsu compiler
     patch('llvm13-fujitsu.patch', when='@13 %fj')
 
+    # patch for missing hwloc.h include for libompd
+    patch('llvm14-hwloc-ompd.patch', when='@14+omp_as_runtime')
```

Comment:
```suggestion
    patch('llvm14-hwloc-ompd.patch', when='@14')
```

This is probably what we want, OpenMP builds either way that variant is to determine whether as a project or a runtime.